docs/pages/index.mdx

```diff
@@ -6,102 +6,81 @@ layout: tocless-doc
+- [DigitalOcean](./try-out-teleport/digitalocean.mdx): Deploy with our 1-Click DigitalOcean droplet.
 
-- [Linux Server](./try-out-teleport/linux-server.mdx): Deploy Open Source
-  Teleport on a Linux Server.
-- [Digital Ocean](./try-out-teleport/digitalocean.mdx): Use our 1-Click Digital
-  Ocean droplet to set up Teleport quickly.
-- [Browser Labs](https://play.instruqt.com/teleport/invite/imz8g1n1xzru): Try Teleport using our guided
-  interactive labs.
-- [Docker Compose Lab](./try-out-teleport/docker-compose.mdx): Try Teleport
-  locally using Docker Compose.
-- [Kubernetes Lab](./try-out-teleport/local-kubernetes.mdx): See how Teleport
-  runs on Kubernetes with this local lab.
+Want more details on the [different editions of Teleport](./choose-an-edition/introduction.mdx)?
 
-## Choose an edition
+## Connect to Teleport
 
-Read one of our Getting Started guides to see how to deploy open source
-Teleport, Teleport Enterprise, or Teleport Cloud.
-
-You can also [compare Teleport editions](./choose-an-edition/introduction.mdx).
-
-- [Open Source Teleport](./try-out-teleport/linux-server.mdx): Learn how to host your own open source Teleport deployment on a standalone Linux server.
-- [Teleport Enterprise](./choose-an-edition/teleport-enterprise/introduction.mdx): Get started with a self-hosted Teleport Enterprise deployment, which gives you more advanced features and full customization.
-- [Teleport Cloud](./choose-an-edition/teleport-cloud/getting-started.mdx): Try our cloud-hosted version for free.
-
-## Configure access
-
-Secure your infrastructure while keeping your engineers productive.
-
-<ScopedBlock scope={["cloud", "enterprise"]}>
-- [Set up SSO](./access-controls/sso.mdx): Configure Teleport's integration with your SSO provider so you can automatically on– and off-board users.
-</ScopedBlock>
-<ScopedBlock scope={["oss"]}>
-- [Set up SSO](./access-controls/sso/github-sso.mdx): Configure Teleport's integration with GitHub so you can automatically on– and off-board users.
-</ScopedBlock>
-- [Define roles](./access-controls/guides/role-templates.mdx): Manage who can access which parts of your infrastructure.
-
-## Add your infrastructure
-
-Use Teleport to provide secure access to all of your infrastructure.
+Already have Teleport running? Start connecting your infrastructure to provide secure access.
 
 - [Server Access](./server-access/getting-started.mdx): Single sign-on, short-lived certificates, and auditing for SSH servers.
 - [Application Access](./application-access/introduction.mdx): Secure access to internal dashboards and web applications.
 - [Kubernetes Access](./kubernetes-access/introduction.mdx): Single sign-on, auditing, and unified access for Kubernetes clusters.
 - [Database Access](./database-access/introduction.mdx): Secure access to SQL and NoSQL databases.
 - [Desktop Access](./desktop-access/introduction.mdx): Secure browser-based access to desktop environments.
 
+## Already familiar with Teleport?
```

Comment:
One challenge that users have when setting up Teleport is the complexity of the software. With the earlier organization of this page, I wanted to leave users with a mental model of what setting up a Teleport deployment looks like, i.e., the sequence of steps they would need to go through. This would help users who have read some of the docs identify where they are in their setup process, and which additional docs they need to read.

This list of links is pretty random to me, and it's hard to discern a sequence of steps for setting up Teleport. I think we should include a more narrative description of the steps a user should follow, e.g., choosing an edition, deploying a cluster, managing access, and connecting resources.

Reply:
Looking at the original again, I think that the text could make it more explicit how the different sections of the landing page relate to one another. It's definitely possible to miss that the headings are supposed to indicate a sequence. Maybe we can include fewer links on the landing page (to make it shorter) but also add more text to describe the way a user should proceed through the docs?